starters/nuxt-2/pages/index.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="home">
+    <div class="home__logo">
+      <img src="/nacelle-logo.svg" />
+    </div>
+    <h1>Welcome to your Nacelle Storefront</h1>
+    <p>Edit <code class="inline-code">pages/index.vue</code> to get started.</p>
+    <p>
+      For help on how to build your storefront,
+      <a href="https://docs.getnacelle.com/nuxt/intro-nuxt.html" target="none"
+        >visit the Nacelle documentation.</a
+      >
+    </p>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'IndexPage'
```

Comment:
This might be clearer:

```suggestion
  name: 'HomePage'
```

It would also be consistent with classes used in the template.